src/store/components/AdminCollectionPage/index.tsx

```diff
@@ -1,59 +1,57 @@
-import { Formik } from 'formik';
 import React from 'react';
+import { Formik } from 'formik';
+import * as Yup from 'yup';
+
+import Config from '../../../config';
+import { history } from '../../../redux_store';
 import { PublicMerchCollection } from '../../../types';
+import { fetchService } from '../../../utils';
+
 import StoreButton from '../StoreButton';
 import StoreColorInput from '../StoreColorInput';
 import StoreHeader from '../StoreHeader';
 import StoreTextInput from '../StoreTextInput';
-import Config from '../../../config';
-import { fetchService, notify } from '../../../utils';
 
 import './style.less';
-import { history } from '../../../redux_store';
 
 interface AdminCollectionPageProps {
   collection?: PublicMerchCollection | undefined;
 }
 
 interface AdminCollectionPageForm {
-  title?: string;
-  themeColorHex?: string;
-  description?: string;
+  title: string;
+  themeColorHex: string;
+  description: string;
```

Comment:
Formik requires all fields to have a value at all times (including for initial values), so I removed the optionals and added the type to the `initialValues` object.